src/angular/styles/_typography.scss

```diff
@@ -553,19 +553,19 @@
     --sbb-secondary-button-background-color-disabled: var(--sbb-color-silver-alpha40);
     --sbb-secondary-button-color-disabled: var(--sbb-color-iron-alpha50);
 
-    // Checkbox
-    --sbb-checkbox-background-color-default: var(--sbb-color-white);
-    --sbb-checkbox-background-color-disabled: var(--sbb-color-milk);
+    // Checkbox & Radiobutton
+    --sbb-selection-box-background-color-default: var(--sbb-color-white);
```

Comment:
Let's change `selection-box` to `selection-container` to align with the CSS class names